torchao/prototype/server/amg_example/README.md

```diff
@@ -0,0 +1,5 @@
+To run this example you need to download the vit_h checkpoint and put it into a local folder named checkpoints
```

Comment:
nit: but prototype is sort of losing its meaning lol. This is closer to an example so mind creating a new examples folder?

Reply:
I'll refactor it to be closer to that in the next PR if that works.

Reply:
I'll just move it into examples right now.